src/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Acquia\Cli\EventListener;
+
+use Acquia\Cli\Exception\AcquiaCliException;
+use League\OAuth2\Client\Provider\Exception\IdentityProviderException;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+
+class ExceptionListener
+{
+
+  public function onConsoleError(ConsoleErrorEvent $event) {
```

Comment:
Tried this based on https://symfony.com/blog/new-in-symfony-4-4-simpler-event-listeners

Reply:
This never seems to get called.